modules/exploits/multi/http/cockpit_cms_rce.rb

```diff
@@ -183,22 +183,23 @@ def report_cred(opts)
   def login(un, pass)
     print_status('Attempting login')
     res = send_request_cgi(
-      'uri' => '/auth/login'
+      'uri' => '/auth/login',
+      'keep_cookies' => true
     )
+
     fail_with(Failure::UnexpectedReply, "#{peer} - Could not connect to the web service") unless res
     fail_with(Failure::UnexpectedReply, "#{peer} - Could not connect to the web service") unless /csfr\s+:\s+"([^"]+)"/ =~ res.body
```

Comment:
I know you didn't change this but is that a typo? `csfr` instead of `csrf`? :eyes:

Reply:
No I think CSFR is possible as well. Just one of those example of acronyms clashing